src/lib/components/EventNetwork.svelte

```diff
@@ -399,42 +551,76 @@
       )
       .style("z-index", 1000);
 
+    const renderTooltip = (d: NetworkNode, pageX: number, pageY: number) => {
```

Comment:
Try turning this function into a Svelte [snippet](https://svelte.dev/docs/svelte/snippet).